docs/can-guides/commitment/recipes/credit-card-advanced/credit-card-advanced.md

```diff
@@ -30,13 +32,15 @@ To use the widget:
    the invalid inputs should be highlighted red, and the _Pay_
    button should become disabled.
 
-__START THIS TUTORIAL BY CLONING THE FOLLOWING JS BIN__:
+__START THIS TUTORIAL BY CLONING THE FOLLOWING CodePen__:
 
-<a class="jsbin-embed" href="https://jsbin.com/woxuhu/2/embed?html,output">
-  Starter Credit Card Guide (Advanced) on jsbin.com
-</a>
+<p class="codepen" data-height="265" data-theme-id="0" data-default-tab="css,result" data-user="cherifGsoul" data-slug-hash="JVPWoZ" style="height: 265px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid black; margin: 1em 0; padding: 1em;" data-pen-title="Credit Card Guide (Starter)">
```

Comment:
I think it’d be better to start with just the JS and Result tabs if possible.

Reply:
Fixed